### PR TITLE
Fix error typo in `testIdWithNull` to `testIdWithFalse`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: tests/AlertTest.php [hunk 1]
```diff
@@ -166,7 +166,19 @@ public function testId(): void
...
-    public function testIdWithNull(): void
+    public function testIdWithEmpty(): void
+    {
+        Assert::equalsWithoutLE(
+            <<<HTML
+            <div class="alert alert-secondary" role="alert">
+            Body
+            </div>
+            HTML,
+            Alert::widget()->id('')->body('Body')->render(),
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply tests/AlertTest.php:177]: https://html.spec.whatwg.org/multipage/dom.html#global-attributes:the-id-attribute-2 The id must be an identifier with a unique value, displaying it without a value is semantically incorrect.

[the remark this answers is not in the document]
Comment (terabytesoftw) [reply tests/AlertTest.php:177]: W3C Validator:
image

[the remark this answers is not in the document]
Comment (vjik) [reply tests/AlertTest.php:177]: Agree. So, may be throw LogicException or RuntimeException on pass empty id?

[the remark this answers is not in the document]
Comment (samdark) [reply tests/AlertTest.php:177]: I'd just avoid rendering it.

**Pull request merged**